packages/customer-account-ui-extensions/src/components/CustomerAccountAction/CustomerAccountAction.ts

```diff
@@ -0,0 +1,22 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import type {ActionComponents} from 'component-set';
```

Comment:
Does this import work? And if it does, should we still import relatively?

I first thought `component-set` would be an npm package. 😅

Reply:
<img width="980" alt="Screenshot 2023-08-18 at 15 13 03" src="https://github.com/Shopify/ui-extensions/assets/12724940/07e424f1-bcda-42b1-8b3e-8aa96308ecbf">

both `../../component-set` and `component-set` works, updating it to relative path now

Reply:
updated in the commit https://github.com/Shopify/ui-extensions/pull/1256/commits/87e2a9838483929ce58859a1c820a06e3ffc42e2